installers/build-certificates-unix.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/sh
+#
+# This is an install-time script that generates the CA/server key/cert pairs,
+# for front/backend channel security.
+# It must be run from the application install directory.
+#
+# Source: https://pki-tutorial.readthedocs.io/en/latest/simple/index.html
+#
+# WARNING: keep in sync with 'installers/build-certificates-win64.bat'
+
+echo Generating certificates.
+openssl version
```

Comment:
There are indications (but no documentation-grade proof) that the `openssl` binary was shipping out-of-the-box at least as early as 2014, and as late as 2016:

- https://bugs.ruby-lang.org/issues/9424#note-32
- https://github.com/rfjakob/gocryptfs/issues/15#issuecomment-265332112

Should we go further than this level of confirmation?